.gitignore

```diff
@@ -12,4 +12,6 @@ coverage
 scripts/mirroring.js
 .docs
 .booster
-*-integration-sandbox
\ No newline at end of file
+*-integration-sandbox
+
+!/packages/cli/resources/**/*
```

Comment:
Why do you added this rule? This folder should be included in the repo anyway, isn't it?

```suggestion
```